docs/topics/http/urls.txt

```diff
@@ -246,8 +246,8 @@ following URL patterns which optionally take a page argument::
     from django.urls import re_path
 
     urlpatterns = [
-        re_path(r'^blog/(page-(\d+)/)?$', blog_articles),                  # bad
-        re_path(r'^comments/(?:page-(?P<page_number>\d+)/)?$', comments),  # good
+        re_path(r'^blog/(page-([0-9]+)/)?$', blog_articles),                  # bad
```

Comment:
I guess an argument could be made for both forms here. I slightly prefer `[0-9]? because it will only allow a single URL to point to the content

Reply:
:balance_scale:  :shrug:

Reply:
"single URL" is a sufficient rationale, IMO.